azext_iot/central/params.py

```diff
@@ -168,12 +221,30 @@ def load_central_arguments(self, _):
             "Use 0 for infinity.",
         )
         context.argument(
-            "module_id", options_list=["--module-id", "-m"], help="Iot Edge Module ID",
+            "module_id", options_list=["--module-id", "-m"], help="Provide Iot Edge Module ID if the device type is IoT Edge.",
+        )
+        context.argument(
```

Comment:
we shouldn't need this, `-d` is used all over the place, i think theres a root level help comment thats currently providing this. Check with @digimaun if this is the right thing to do

Reply:
Do you mean the ["--device-id", "-d"]?

Reply:
yes, the way these parameters work is that the help is usually pulled from some base command. Duplicating this help text/having multiple help texts can be confusing and add maintenance debt.

We should just add all the information as a `long-summary:` to the root level --device-id field

Reply:
I see, followed the excel file to changed all, also feel wired that we have to define for each different command that device-id description "like id of device to delete, id of device to show detail" are redundant. we should check with Ankit for the documents.
![image](https://user-images.githubusercontent.com/14871527/101098329-94471980-3577-11eb-83a6-e599037e8078.png)

Reply:
sounds good, I'll hit waiting until this is resolved. Feel free to add me to any meetings if I'm needed

Reply:
Right, we want to reduce re-defining args as much as we can. It's great when we can try to align common args and their descriptions across our IoT services. 

However being cognizant that there are cases where existing argument descriptions may be too general for the command group or command - in that case we can provide more specific guidance right in the help description of the command group or command. The `long-summary` is a good candidate for these details like @prbans suggested.

Then if there _really_ is value/clarity for redefining the arg then we can certainly do it - but again try and position in a common way. For example, arbitrarily choosing this, if there are additional description clarifications for `--device-id` for the `az iot central diagnostics validate-messages`  can they be applied to `az iot central diagnostics` instead?

Reply:
@digimaun our PM made two different help strings for --device-id in `az iot central diagnostics validate-messages `and `az iot central diagnostics validate-properties`, let me contact with PM and see if we can make the description simpler, as when user use the command, they already know --device-id means for which action, seems not required to describe on help string.

Reply:
@prbans @digimaun checked with our PM, they agree that we could unified under `az iot central` for --device-id, as it need a extra description for how to get the device id from central UX, code change was checked in now, please let me know if you have any other comments.